swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -8,15 +8,43 @@ active: provide-values-to-configurable-variables
 intro: You can provide values for configurable variables through multiple methods, as described below.
 ---
 
->**Note:** If the configuration values are given in multiple ways, they will be overridden in the following precedence order when retrieving them.
+>**Note:** If the configuration values are given in multiple ways, they will be overridden in the following decreasing order of precedence when retrieving them.
```

Comment:
This seems unclear. how about 
**Note:** If the configuration values are given in multiple ways, they will be overridden in the following order when retrieving them.